### ISSUE TITLE
istioctl proxy-status not showing patch version

### ISSUE THREAD

**Issue by Stono** [istio/istio#8244]
Just upgraded to `1.0.1`, including all side cars, but they're still showing as `1.0.0`:

eg:
```
❯ istioctl proxy-status
PROXY                                                                    CDS        LDS        EDS               RDS        PILOT                            VERSION
elastic-heartbeat-java-app-b4bf94bd-mbp46.elastic-heartbeat-java-app     SYNCED     SYNCED     SYNCED (100%)     SYNCED     istio-pilot-7784df98d8-dpmzg     1.0.0
ingress-nginx-external-controller-6fc57cc9c4-flxmp.ingress-nginx         SYNCED     SYNCED     SYNCED (100%)     SYNCED     istio-pilot-7784df98d8-dpmzg     1.0.0
ingress-nginx-internal-controller-54fbd99b5f-bcldg.ingress-nginx         SYNCED     SYNCED     SYNCED (100%)     SYNCED     istio-pilot-7784df98d8-dm62z     1.0.0
mike-istio-test-pod.default                                              SYNCED     SYNCED     SYNCED (100%)     SYNCED     istio-pilot-7784df98d8-dm62z     1.0.0
oauth-redis-proxy-5cf64d7bf4-9n89t.ingress-nginx                         SYNCED     SYNCED     SYNCED (100%)     SYNCED     istio-pilot-7784df98d8-dm62z     1.0.0
oauth-redis-proxy-5cf64d7bf4-h479l.ingress-nginx                         SYNCED     SYNCED     SYNCED (100%)     SYNCED     istio-pilot-7784df98d8-dpmzg     1.0.0
oauth2-proxy-79b9d6644d-fwbzv.ingress-nginx                              SYNCED     SYNCED     SYNCED (100%)     SYNCED     istio-pilot-7784df98d8-dm62z     1.0.0
platform-testing-6b5f989b5-hv9z9.platform-testing                        SYNCED     SYNCED     SYNCED (100%)     SYNCED     istio-pilot-7784df98d8-dpmzg     1.0.0
shippr-simple-5c9d9b7b68-n27hr.shippr-simple                             SYNCED     SYNCED     SYNCED (100%)     SYNCED     istio-pilot-7784df98d8-dpmzg     1.0.0
```

Purely cosmetic I guess.

**Comment by sdake** [comment]
@Stono I'll take a look.

Cheers
-steve

**Comment by sdake** [comment]
@Stono thanks for the report.  This has been merged to the release-1.0 branch and will be available in the next z stream (Istio 1.0.2).  The problem did not exist in master.

**Comment by Stono** [comment]
@sdake fyi see #15094 - where this problem still exists in `1.2.0`
